distribution/lib/Standard/Base/0.0.0-dev/src/Data.enso

```diff
@@ -189,6 +189,8 @@ read_text path=(Missing_Argument.throw "path") (encoding : Encoding = Encoding.d
      files from the subdirectories. If set to `False` (the default), only the
      immediate children of the listed directory are considered.
 
+   ? Name Filter Rules
+
    The `name_filter` can contain the following special characters:
    - `"?"` - which matches a single filename character (so it will not match a
      `"/"`).
```

Comment:
Shouldn't the text following `? Name Filter Rules` heading be indented 2 spaces? I think that was the format we used to align to.